cabot/cabotapp/models.py

```diff
@@ -164,13 +164,13 @@ class Meta:
         'MatterMostInstance',
         null=True,
         blank=True,
-        help_text='MatterMost instance to send alerts to (can be None if MatterMost alerts disabled).'
+        help_text='Mattermost instance to send alerts to (can be blank if Mattermost alerts are disabled).'
     )
     mattermost_channel_id = models.CharField(
         null=True,
         blank=True,
         max_length=100,
```

Comment:
nit:should we make this one 32 also if that's the max length?

Reply:
26 seems to be the length of a channel ID I checked, but it's not documented anywhere. Figured I'd leave a little extra just in case they change something at some point.